CSS/index.css

```diff
@@ -33,7 +33,7 @@
   width: 100%;
   text-align: center;
   padding: 15px 0;
-  font-size: 18px;
+  font-size: 1px;
```

Comment:
Your code works perfectly but this is the line that made the content of your links barely visible.